packages/ui/src/1_atoms/Link/Link.test.tsx

```diff
@@ -1,22 +1,24 @@
-import { render, screen } from '@testing-library/react';
+import { render } from '@testing-library/react';
 
 import React from 'react';
 
 import { Link } from './Link';
 
-test('renders Link with action id', () => {
-  render(
+test.only('renders Link with action id', () => {
```

Comment:
```suggestion
test('renders Link with action id', () => {
```

Remove this as currently only this test will run